pkgs/development/python-modules/py_stringmatching/default.nix

```diff
@@ -0,0 +1,29 @@
+
+{ lib
+, buildPythonPackage
+, numpy
+, six
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "py_stringmatching";
+  version = "0.4.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0rjsx7iipn6svki21lmsza7b0dz9vkgmix696zryiv7gkhblqyb4";
+  };
+
+  # No tests in archive
```

Comment:
Wrong! Just look in `py_stringmatching/tests/`.
Please enable tests and use `nosetests` in the `checkPhase`.